server/routes/makeAReferral/completion-deadline/completionDeadlineForm.test.ts

```diff
@@ -39,4 +42,97 @@ describe('CompletionDeadlineForm', () => {
       })
     })
   })
+  describe('for a sent referral', () => {
+    it('no errors for valid data', async () => {
+      const request = TestUtils.createRequest({
+        'completion-deadline-year': '2021',
+        'completion-deadline-month': '09',
+        'completion-deadline-day': '12',
+        'reason-for-change': 'reason',
+      })
+
+      const data = await new CompletionDeadlineForm(request, true).data()
+      expect(data.paramsForUpdate).toEqual({
+        draftReferral: { completionDeadline: '2021-09-12' },
+        reasonForUpdate: 'reason',
+      })
+    })
+
+    it('no errors for valid data', async () => {
+      const request = TestUtils.createRequest({
+        'completion-deadline-year': '2021',
+        'completion-deadline-month': '09',
+        'completion-deadline-day': '12',
+        'reason-for-change': 'reason',
+      })
+
+      const data = await new CompletionDeadlineForm(request, true).data()
+      expect(data.paramsForUpdate).toEqual({
+        draftReferral: { completionDeadline: '2021-09-12' },
+        reasonForUpdate: 'reason',
+      })
+    })
+    it('returns an error if reason for change is invalid but completion date is valid', async () => {
+      const request = TestUtils.createRequest({
+        'completion-deadline-year': '2021',
+        'completion-deadline-month': '09',
+        'completion-deadline-day': '12',
+        'reason-for-change': '',
+      })
+
+      const data = await new CompletionDeadlineForm(request, true).data()
+      expect(data.error).toEqual({
+        errors: [
+          {
+            errorSummaryLinkedField: 'reason-for-change',
+            formFields: ['reason-for-change'],
+            message: 'Reason for change cannot be empty',
+          },
+        ],
+      })
+    })
+    it('returns an error if completion date is valid but reason for change is invalid', async () => {
```

Comment:
I believe the the test name is completion date is invalid and reason of change is valid?